src/NHSD.BuyingCatalogue.SolutionLists.API/ViewModels/CapabilityReferenceViewModel.cs

```diff
@@ -0,0 +1,9 @@
+﻿using NHSD.BuyingCatalogue.SolutionLists.Contracts;
+
+namespace NHSD.BuyingCatalogue.SolutionLists.API.ViewModels
+{
+    public sealed class CapabilityReferenceViewModel : ICapabilityReference
```

Comment:
We've used an ISet<> as part of the data type. This will do an equality check, meaning the Equals & GetHashCode will need to be implemented within this class.

Reply:
👍